lib/fastlane/plugin/poesie/actions/poesie_action.rb

```diff
@@ -70,6 +70,11 @@ def self.available_options
             env_name: "PROJECT_STRING_FILE_NAME",
             description: "Name of a localized .strings file. Default is Localizable.strings",
             optional: true,
+            type: String),
+          FastlaneCore::ConfigItem.new(key: :strings_path,
+            env_name: "PROJECT_STRING_PATH",
+            description: "Path of a localized .strings file",
```

Comment:
Možno bych tady dal neco jako `Path to a folder where poesie will search for .string files`. Pretože aj mna samotného zmiatlo, že vlastne len upresnuješ kde ten script bude hladať .string súbory.